whois-api/src/test/java/net/ripe/db/whois/api/rest/WhoisRestServiceTestIntegration.java

```diff
@@ -1835,6 +1835,67 @@ public void create_succeeds() {
         assertThat(whoisResources.getTermsAndConditions().getHref(), is(WhoisResources.TERMS_AND_CONDITIONS));
     }
 
+
```

Comment:
Add a test case for UPDATE flat (non-hierarchical) AS-SET with `source: TEST-NONAUTH`